test/extended/idling/idling.go

```diff
@@ -239,7 +239,7 @@ var _ = g.Describe("[sig-network-edge][Feature:Idling]", func() {
 		})
 	})
 
-	g.Describe("Unidling", func() {
+	g.Describe("Unidling [apigroup:apps.openshift.io]", func() {
```

Comment:
`route.openshift.io` as well (both apps and route are used in `testdata/idling-echo-server.yaml` (set through `echoServerFixture`)

Reply:
Also `with a single service and DeploymentConfig` (line 218) is using `route.openshift.io` (through `testdata/idling-echo-server-rc.yaml`)

Reply:
Done